query_server/sqllogicaltests/cases/ddl/alter_drop_after.slt

```diff
@@ -1,32 +1,32 @@
 #statement ok 
```

Comment:
Why pull request to upgrade dependencies contains the change of sqllogicaltests?

Reply:
a tenant with the same name is set in other tests, it may cause error